var/spack/repos/builtin/packages/aom/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2019 Trinity College Dublin and other Spack Project
+# Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Aom(CMakePackage):
+    """Alliance for Open Media"""
```

Comment:
@smcgrat this all looks good but could you add a bit to the description, like

> Alliance for Open Media's AV1 Codec Library

(or at least that's what I gather from the site). Anyhow, it doesn't have to be a lot more, just provide some info on what the package is for.

Reply:
Thanks. I've added a more descriptive license information. Not that the library's website is very descriptive on what it does!
Sean